### PR TITLE
Swap Slotmachine with avWebQLApi

### PR DESCRIPTION
**Pull request by nylon22**
Leaving this as a draft until related changes to avWebQLApi are released

### CODE REVIEW

File: packages/spaces/src/Spaces.js [hunk 1]
```diff
@@ -21,21 +21,24 @@ export const getAllSpaces = async (
...
-  const unionedSpaces = _spaces.concat(spaces.spaces);
+  const {
+    pageInfo: { itemCount, currentPage, perPage },
+  } = configurationPagination;
+
+  const unionedSpaces = _spaces.concat(configurationPagination.items);
 
-  if (totalCount > page * perPage) {
-    const vars = { ...variables, page: page + 1 };
+  if (itemCount > currentPage * perPage) {
```
Comment (chrishavekost) [inline packages/spaces/src/Spaces.js:41]: `pageInfo` contains a `hasNextPage` boolean we could use here for readability, but I'm fine with this either way

File: packages/spaces/src/Spaces.js [hunk 2]
```diff
@@ -196,34 +200,98 @@ Spaces.propTypes = {
...
+            url
           }
-          url
+
         }
       }
     }
   `,
-  variables: { types: ['space'] },
+  variables: { types: ['PAYERSPACE'] },
```
Comment (chrishavekost) [inline packages/spaces/src/Spaces.js:295]: Do we need to add the other configuration types as variables here?

File: packages/payer-logo/PayerLogo.js [hunk 3]
```diff
@@ -1,27 +1,33 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useEffectAsync } from '@availity/hooks';
-import { avSlotMachineApi } from '@availity/api-axios';
-import get from 'lodash/get';
+import { avWebQLApi } from '@availity/api-axios';
+import get from 'lodash.get';
```
[the remark this answers is not in the document]
Comment (nylon22) [reply packages/payer-logo/PayerLogo.js]: good catch 81ccf0c

File: packages/spaces/src/Spaces.js (hunk 1 above)
Comment (nylon22) [reply packages/spaces/src/Spaces.js:41]: good idea. i left as a TODO for now f70a632

File: packages/spaces/src/Spaces.js (hunk 2 above)
Comment (nylon22) [reply packages/spaces/src/Spaces.js:295]: not for the default. devs who implement this package can override that prop.

Comment (chrishavekost) [reply packages/spaces/src/Spaces.js:295]: Gotcha, thanks for clarifying

**Review by gregmartDOTin — APPROVED** [review]
LGTM

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://61003922a835de528e60b7c9--condescending-leavitt-66e607.netlify.app

**Review by chrishavekost — APPROVED** [review]

**Pull request merged**